### ISSUE TITLE
Recruit volunteers for team open roles

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#3222]
### Dependency
- [ ] need to recruit new team members to website team

### Overview
We need a data analyst to work on the open issues we have so that we can move the project forward.

### Action Items
- [ ] Copy the template X.03 (where X is the number of the practice area) from the practice area for the role (see practice areas below in resources) into a comment.  (e.g., under the 1 Data Science dropdown, the template would 1.03)
- [ ] customize the comment for the new open role, including adding your name as the person to contact.
- [ ] paste the comment you just customized to a comment on https://github.com/hackforla/website/issues/6987
- [ ] on https://github.com/hackforla/website/issues/6987, if there is a label `recruiting: no`, remove that label and add the label `recruiting: yes`
- [ ] on #6987 change 'Status' by removing `~` from no line, and add `~` to yes line
- [ ] add a link from that new comment to the comment on this issue
- [ ] Post a link to the comment from the specific open role issue to the related Community of Practice Slack Channel
- [ ] 
- [ ] 

- [ ] keep yourself assigned to this issue until the role is filled
- [ ] check our team channel for people reaching out

#### once filled
- [ ] hide the comment on the https://github.com/hackforla/website/issues/6987 issue
- [ ] on https://github.com/hackforla/website/issues/6987, if there are no other open roles for data science, remove the  `recruiting: yes`, and add the label `recruiting: no`
- [ ] on #6987 change 'Status' by removing `~` from yes line, and add `~` to no line
- [ ] hide the comment on this issue 
- [ ] put issue back in icebox
- [ ] add the dependency label
- [ ] uncheck the dependency

### Resources/Instructions

<details><summary>1 Data Science</summary>

- 1.01 https://github.com/hackforla/website/issues/6987
- 1.02 Slack message to post role
   ```
   Hi, the website team has a open role for Data Scientist.  Please see details here, including how to get in touch with us
   ```
- 1.03  Draft post for Data role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: 

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#product-management](https://hackforla.slack.com/archives/CJY4R382X)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```

</p>
</details> 

<details><summary>Details</summary>
<p>

- 2.01 https://github.com/hackforla/website/issues/6987
- 2.02 Slack message to post role
   ```
   Hi, the website team has a open role for Product Management.  Please see details here, including how to get in touch with us
   ```
- 2.03  Draft post for Data role

```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: We are looking for a Product Manager/Project Manager for our team

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#product-management](https://hackforla.slack.com/archives/CJY4R382X)
- Slack name of person to contact in the channel: see who this issue is assigned to and @message them in the channel above

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
   ```

</p>
</details>

**Comment by ExperimentsInHonesty** [comment]
@JessicaLucindaCheng @SAUMILDHANKAR If we have some issues for Data Scientists or we think some issues are going to be made, please add any details here that would help

**Comment by JessicaLucindaCheng** [comment]
Will find the issues related needing a Data Scientist
Availability: 1 hour
ETA: Monday, November 14, 2022

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty 
> @JessicaLucindaCheng @SAUMILDHANKAR If we have some issues for Data Scientists or we think some issues are going to be made, please add any details here that would help

I couldn't find any open issues or issues to be made related to having a Data Scientist. However, if I remember correctly, the topic of recruiting a data scientist came up when we had a team discussion about getting some stats on how far people get through the matriculation process, including 
 - how far they have gotten through the prework, 
 - good first issue, 
 - good second issue, etc. 

In particular, I remember there was interest in figuring out 
- where people stopped in the matriculation process and why
- how that differed from people that completed matriculation

Also, I remember from the discussion that before we recruit a data scientist we would need to gather the data for them to analyze. This would include but not limited to:
- Labeling each prework issue for each person with how far they got. For example, if they completed their good first issue and added their ETA and availability correctly, we could add the following labels to their prework: `a:cgfi` to stand for "achievement: completed good first issue" and `a:ae&ac` to stand for "achievement: added ETA and availability correctly". 
- After labeling all the preworks, creating some kind of automation to go through all the prework issues and gathering/aggregating the data provided by the labels into a table or spreadsheet.
- After gathering/aggregating the data, then a data scientist could analyze it.
- Also, after gathering/aggregating the data, possibly have UX researchers interview or ask people to see why they stopped before completing matriculation. (I'm not sure if this would happen before or in parallel with a data scientist analyzing the data.)

Since all the above was from my memory, I could be remembering incorrectly why we needed a data scientist. Let me know if you have any questions about any of the above.

**Comment by ExperimentsInHonesty** [comment]
This issue is being replaced by new recruitment issue that uses the new project boards.

**Comment by ExperimentsInHonesty** [comment]
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: We are looking for a Product Manager/Project Manager for our team

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#product-management](https://hackforla.slack.com/archives/CJY4R382X)
- Slack name of person to contact in the channel: see what this issue is assigned to

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)

**Comment by Rabia2219** [comment]
### Dependency
- [ ] need to recruit new team members to website team

### Overview
We need a volunteer to work on the open issues we have so that we can move the project forward.

### Action Items
- [ ] find the related practice area for the open role from the resources below (X is the number of the practice area, e.g. Data Science is 1)
- [ ] copy the template from resource X.03 from the practice area into a comment (e.g. for Data Science, the template would 1.03)
- [ ] customize the comment for the new open role, including adding your name as the person to contact
- [ ] paste the contents of the comment you just customized to a new comment on the issue in resource X.01 (e.g. for a data scientist, the issue would be Open Role: Data Scientist)
- [ ] add the link of that new comment to the comment you customized on this issue, using resource X.04
- [ ] on issue in resource X.01, if there is a label `recruiting: no`, remove that label and add the label `recruiting: yes`
- [ ] on the issue in resource X.01 change 'Status' by removing `~` from no line, and add `~` to yes line
- [ ] post a link to the new comment from the issue in resource X.01 to the related Community of Practice Slack Channel, using resource X.02
- [ ] keep yourself assigned to this issue until the role is filled
- [ ] check our team channel for people reaching out

#### Once Filled
- [ ] hide the comment on the issue in resource X.01
- [ ] on the issue in resource X.01, if there are no other open roles for data science, remove the  `recruiting: yes`, and add the label `recruiting: no`
- [ ] on the issue in X.01 change 'Status' by removing `~` from yes line, and add `~` to no line
- [ ] hide the comment on this issue 
- [ ] put issue back in icebox
- [ ] add the dependency label
- [ ] uncheck the dependency

### Resources/Instructions

<details><summary>1 Data Science</summary>

- 1.01 https://github.com/hackforla/website/issues/6987
- 1.02 Slack message to post role
   ```
   Hi, the website team has an open role for Data Scientist.  Please see details [here](https://github.com/hackforla/website/issues/6987), including how to get in touch with us
   ```
- 1.03  Template for Data Science role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 1.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on #6987  

```
</p>
</details> 

<details><summary>2 Data Analysis</summary>

- 2.01 Open Role: Data Analyst ISSUE NUMBER
- 2.02 Slack message to post role
   ```
   Hi, the website team has an open role for Data Analyst.  Please see details [here](Open Role: Data Analyst ISSUE URL), including how to get in touch with us
   ```
- 2.03  Template for Data Analysis role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 2.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on Open Role: Data Analyst ISSUE NUMBER

```
</p>
</details> 

<details><summary>3 Data Analysis: Visualization</summary>

- 3.01 Open Role: Data Analyst: Visualization ISSUE NUMBER
- 3.02 Slack message to post role
   ```
   Hi, the website team has an open role for Data Analyst: Visualization.  Please see details [here](Open Role: Data Analyst: Visualization ISSUE URL), including how to get in touch with us
   ```
- 3.03  Template for Data Analysis: Visualization role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 3.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on Open Role: Data Analyst: Visualization ISSUE NUMBER

```
</p>
</details>

**Comment by Rabia2219** [comment]
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: The website team is looking for a data scientist to work on 

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://hackforla.slack.com/archives/CJY4R382X)
- Slack name of person to contact in the channel: [Rabia Shaikh]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)

**Comment by Rabia2219** [comment]
Hi, the website team has an open role for Data Scientist.  Please see details [here](#6987), including how to get in touch with us

**Comment by Rabia2219** [comment]
<details><summary>1 Data Science</summary>

- 1.01 https://github.com/hackforla/website/issues/6987
- 1.02 Slack message to post role
   ```
   Hi, the website team has an open role for Data Scientist.  Please see details [here](https://github.com/hackforla/website/issues/6987), including how to get in touch with us
   ```
- 1.03  Template for Data Science role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 1.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on #6987  

```
</p>
</details> 

<details><summary>2 Data Analysis</summary>

- 2.01 Open Role: Data Analyst ISSUE NUMBER
- 2.02 Slack message to post role
   ```
   Hi, the website team has an open role for Data Analyst.  Please see details [here](Open Role: Data Analyst ISSUE URL), including how to get in touch with us
   ```
- 2.03  Template for Data Analysis role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 2.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on Open Role: Data Analyst ISSUE NUMBER

```
</p>
</details> 

<details><summary>3 Data Analysis: Visualization</summary>

- 3.01 Open Role: Data Analyst: Visualization ISSUE NUMBER
- 3.02 Slack message to post role
   ```
   Hi, the website team has an open role for Data Analyst: Visualization.  Please see details [here](Open Role: Data Analyst: Visualization ISSUE URL), including how to get in touch with us
   ```
- 3.03  Template for Data Analysis: Visualization role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 3.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on Open Role: Data Analyst: Visualization ISSUE NUMBER

```
</p>
</details> 




<details><summary>4 UX Community of Practice Co-Leadership</summary>

- 4.01 Open Role: UX Community of Practice Co-Lead ISSUE NUMBER
- 4.02 Slack message to post role
   ```
   Hi, the website team has an open role for UX Community of Practice Co-Lead.  Please see details [here](Open Role: UX Community of Practice Co-Lead ISSUE URL), including how to get in touch with us
   ```
- 4.03  Template for UX Community of Practice Co-Leadership role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 4.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on Open Role: UX Community of Practice Co-Lead ISSUE NUMBER

```
</p>
</details> 

<details><summary>5 Design Leadership</summary>

- 5.01 Open Role: Design Lead ISSUE NUMBER
- 5.02 Slack message to post role
   ```
   Hi, the website team has an open role for Design Lead.  Please see details [here](Open Role: Design Lead ISSUE URL), including how to get in touch with us
   ```
- 5.03  Template for Design Leadership role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 5.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on Open Role: Design Lead ISSUE NUMBER

```
</p>
</details> 
<details><summary>6 UX/Product Design</summary>

- 6.01 Open Role: UX/Product Designer ISSUE NUMBER
- 6.02 Slack message to post role
   ```
   Hi, the website team has an open role for UX/Product Designer.  Please see details [here](Open Role: UX/Product Designer ISSUE URL), including how to get in touch with us
   ```
- 6.03  Template for UX/Product Design role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 6.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on Open Role: UX/Product Designer ISSUE NUMBER

```
</p>
</details> 

<details><summary>7 UI/Visual Design</summary>

- 7.01 Open Role: UI/Visual Designer ISSUE NUMBER
- 7.02 Slack message to post role
   ```
   Hi, the website team has an open role for UI/Visual Designer.  Please see details [here](Open Role: UI/Visual Designer ISSUE URL), including how to get in touch with us
   ```
- 7.03  Template for UI/Visual Design role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 7.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on Open Role: UI/Visual Designer ISSUE NUMBER

```
</p>
</details> 

<details><summary>8 UX Research</summary>

- 8.01 Open Role: UX Researcher ISSUE NUMBER
- 8.02 Slack message to post role
   ```
   Hi, the website team has an open role for UX Researcher.  Please see details [here](Open Role: UX Researcher ISSUE URL), including how to get in touch with us
   ```
- 8.03  Template for UX Research role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 8.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on Open Role: UX Researcher ISSUE NUMBER

```
</p>
</details> 

<details><summary>9 UX Research Team Leadership</summary>

- 9.01 Open Role: UX Research Team Lead ISSUE NUMBER
- 9.02 Slack message to post role
   ```
   Hi, the website team has an open role for UX Research Team Lead.  Please see details [here](Open Role: UX Research Team Lead ISSUE URL), including how to get in touch with us
   ```
- 9.03  Template for  UX Research Team Leadership role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 9.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on Open Role: UX Research Team Lead ISSUE NUMBER

```
</p>
</details>

<details><summary>10 UX Program Design Management</summary>

- 10.01 Open Role: UX Program Design Manager ISSUE NUMBER
- 10.02 Slack message to post role
   ```
   Hi, the website team has an open role for UX Program Design Manager.  Please see details [here](Open Role: UX Program Design Manager ISSUE URL), including how to get in touch with us
   ```
- 10.03  Template for UX Program Design Management role
```
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)
**Project Name**: Hack for LA website

**Volunteer Opportunity**: [Replace with CUSTOM RECRUITMENT MESSAGE]

**Duration**: On-going

**Meeting Times**: Sun 10 am to 12 pm

**Who to communicate your interest to**
- Slack channel link: [#hfla-site](https://app.slack.com/client/T04502KQX/C4UM52W93)
- Slack name of person to contact in the channel: [Replace with YOUR SLACK NAME]

[Read more about the project](https://www.hackforla.org/projects/website)

#### When we meet:
   - Sundays at 10:00 -11:00am PST (All Team Meeting)
```
- 10.04 Link of comment on open role issue
```
- Open role [posting](Replace with LINK TO NEW COMMENT) on Open Role: UX Program Designer ISSUE NUMBER

```
</p>
</details>

**Comment by Rabia2219** [comment]
<details><summary>11 Front End Development</summary>

</details>

<details><summary>12 Engineering Community of Practice Leadership</summary>

</details>

<details><summary>13 Team Tech Leadership</summary>

</details>

<details><summary>14 Back End Development</summary>

</details>

<details><summary>15 Full Stack Development</summary>

</details>

<details><summary>16 Quality Assurance</summary>

</details>

**Comment by Rabia2219** [comment]
<details><summary>17 DevOps Community Leadership</summary>

</details>

<details><summary>18 Data Engineering</summary>

</details>

<details><summary>19 Database Architecture</summary>

</details>

<details><summary> 20 Site Reliability Engineering</summary>

</details>

* had cards for Product Manager and Back End Developer

**Comment by Rabia2219** [comment]
<details><summary>21 Product Management</summary>

</details>

<details><summary>22 Business Analysis</summary>

</details>

<details><summary>23 Product Ownership</summary>

</details>

<details><summary>24 Lead Product Management</summary>

</details>

<details><summary>25 Special Projects Coordination</summary>

</details>

<details><summary>26 Project Management</summary>

</details>

**Comment by Rabia2219** [comment]
<details><summary>27 Marketing & Communications Internship</summary>

</details>

<details><summary>28 Marketing & Communications Coordination</summary>

</details>

<details><summary>29 Marketing & Communications Management</summary>

</details>

<details><summary>30 Marketing & Communications Directorship</summary>

</details>

<details><summary>31 Content Strategist (UX Writer)</summary>

</details>

**Comment by Rabia2219** [comment]
<details><summary>32 Institutional Giving Management</summary>

</details>

<details><summary>33 Individual Giving Management</summary>

</details>

<details><summary>34 Project Fundraising</summary>

</details>

**Comment by ExperimentsInHonesty** [comment]
#### latest draft

### Dependency
- [ ] need to recruit new team members to website team

### Overview
We need volunteer(s) to work on the open issues we have so that we can move the project forward.

### Action Items
#### Creating a new role post
- [ ] Copy the template post from the Community of Practice WIKI, Open Roles Templates, Secondary Issue (Resource 1.01)
- [ ] Paste the template in a comment below
- [ ] Customize the comment to reflect your open role
- [ ] Open a new issue by using the Open Role template (Resource 1.02)
- [ ] Copy and Paste your customized comment to the new issue
- [ ] Add the label for the role that you are recruiting for
- [ ] Submit the new issue
- [ ] Assign yourself to the new issue
- [ ] Add the issue to the HfLA: Open Roles project board (Resource 1.03)
- [ ] add the link of the new issue to the related comment on this issue 
- [ ] keep yourself assigned to this issue until the role is filled

#### Marketing the new role post
-  Post a link to the new issue in the Slack channel for the related Practice Area
   - [ ] Copy the Slack message  (Resource 1.04)
   - [ ] Copy the message into the Slack Channel for the relevant Community of Practice (The Slack channels can be found on Resource 1.01)
     - [ ] Replace the text `[Replace with LINK TO NEW ISSUE]` with link to the issue
   - [ ] check your team Slack channel for people reaching out after every onboarding and at least every other day

#### Once Filled
- [ ] Hide the comment on this issue for the role you just filled
- [ ] Add the following comment to the role posting issue 
  ```
  This role has been filled by @[Replace with GITHUB HANDLE OF PERSON WHO IS TAKING ROLE]
  ```
- [ ] Close the role posting issue
- [ ] Unassign yourself from this issue
- If there are no open roles on this issue (i.e., if there are open roles, there will be unclosed comments)
   - [ ] Add the label: Dependency and uncheck the dependency on top of this issue
   - [ ] Move this issue to the icebox, on the project board

### Resources/Instructions
- 1.01 [Community of Practice WIKI, Open Roles Templates, Secondary Issue](https://github.com/hackforla/communities-of-practice/wiki/Open-Roles-Templates#secondary-issue)
- 1.02 [Replace with OPEN ROLE TEMPLATE]
- 1.03 [HfLA: Open Roles board](https://github.com/orgs/hackforla/projects/67)
- 1.04 Copy to post in CoP Slack Channels when there is an open role
   ```
   Hi, the [Replace with NAME OF TEAM] has a volunteer posting, which includes details of how to get in touch with us if you are interested. [Replace with LINK TO NEW ISSUE]
   ````

**Comment by ExperimentsInHonesty** [comment]
### labels to add to the recruitment issue:
- [x] feature: recruiting

### delete recruiting yes no labels

- [x] TDM
- [x]  VRMS
- [x] Website

### Add the Website team image at top to its template
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)

**Comment by ExperimentsInHonesty** [comment]
### Title
HfLA website: Open Role for: Back End Developer

### Issue
- https://github.com/hackforla/website/issues/6987

### Body
![image](https://user-images.githubusercontent.com/37763229/162992303-b0fae60e-1886-45ce-93c1-d1a27a909327.png)

**Project Name**: Hackforla.org Website

**Name of Role**: Back End Developer

**Volunteer Opportunity**: Need someone with experience with GitHub actions automation, Google AppsScript, and working with REST and GraphQL calls. The developer will work very closely with the VRMS and CivicTechJobs teams, who manages the data we receive. **Note:** The Website project is unique in that it is hosted on GitHub Pages. This means we do not have an official backend.

**Duration**: Open-ended

**Meeting times**:  See project page below

**Who to communicate your interest to**
- Slack channel link: #hfla-site
- Slack name of person to contact in the channel: see who this issue is assigned to and @ message them in the channel above

[Read more about the project](https://www.hackforla.org/projects/website)

**Resources:**
- [Welcome to the HackforLA.org (HfLA) Website Wiki!](https://github.com/hackforla/website/wiki)
- [VRMS](https://www.hackforla.org/projects/vrms)
